src/errors.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-cycle
```

Comment:
Don't ever disable (// eslint-disable-next-line import/no-cycle) this warning. Rearrange the imports so they dont cause a cycle